TabloidCLI/Repositories/PostRepository.cs

```diff
@@ -131,12 +132,44 @@ OUTPUT INSERTED.Id
 
         public void Update(Post post)
         {
-            throw new NotImplementedException();
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"UPDATE POST 
+                                           SET Title = @title,
+                                               URL = @url,
+                                               PublishDateTime =                                   @publishdatetime
+                                               Author.Id = @authorid
+                                               Blog.Id = @blogid
+                                         WHERE id = @id";
+
+                    cmd.Parameters.AddWithValue("@title", post.Title);
+                    cmd.Parameters.AddWithValue("@url", post.Url);
+                    cmd.Parameters.AddWithValue("@publishdatetime", post.PublishDateTime);
+                    cmd.Parameters.AddWithValue("@authorid", post.Author.Id);
+                    cmd.Parameters.AddWithValue("@id", post.Blog.Id);
+                    cmd.Parameters.AddWithValue("@id", post.Id);
+
+                    cmd.ExecuteNonQuery();
+                }
+            }
         }
 
         public void Delete(int id)
         {
-            throw new NotImplementedException();
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"DELETE FROM Post 
```

Comment:
Have you tested this with a tagged post?

Reply:
I have not. Im assuming I would need to refactor to keep in consideration for a tag? We havent implemented adding a tag to a post just yet